tests/zfs_test.go

```diff
@@ -0,0 +1,51 @@
+package mos_test
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/spectrocloud/peg/matcher"
+)
+
+var _ = Describe("kairos zfs test", Label("zfs"), func() {
```

Comment:
plans to attach this test to the CI? I think we can give this at least for a flavor. or maybe put as part of our acceptance tests, as a separate block?

Reply:
I like the idea of running it for at least one flavor, I think acceptance tests make sense